src/Test/Utilities/Portable/Compilation/TestOperationVisitor.cs

```diff
@@ -1122,6 +1122,28 @@ public override void VisitDeclarationPattern(IDeclarationPatternOperation operat
             Assert.Empty(operation.Children);
         }
 
+        public override void VisitRecursivePattern(IRecursivePatternOperation operation)
```

Comment:
Please add calls to the rest of the properties of the operation to ensure they don't throw. It looks like most of the rest of the pattern verifiers here also need this change.